QuizApp/App/Protocols/CoordinatorProtocol.swift

```diff
@@ -0,0 +1,7 @@
+import UIKit
+
+protocol CoordinatorProtocol {
+
+    @discardableResult func showLogin() -> UIViewController
```

Comment:
Coordinator functions should not return anything.

Reply:
Refactored.